src/engine/cancun.md

```diff
@@ -74,6 +85,21 @@ This structure has the syntax of [`PayloadAttributesV2`](./shanghai.md#payloadat
 - `withdrawals`: `Array of WithdrawalV1` - Array of withdrawals, each object is an `OBJECT` containing the fields of a `WithdrawalV1` structure.
 - `parentBeaconBlockRoot`: `DATA`, 32 Bytes - Root of the parent beacon block.
 
+### StatelessPayloadStatusV1
+
+This structure contains the result of processing a payload. The fields are encoded as follows:
+
+- `status`: `enum` - `"VALID" | "INVALID"`
+- `stateRoot`: `DATA|null`, 32 Bytes - the state root of the most recent *valid* block in the branch defined by payload and its ancestors
+- `receiptsRoot`: `DATA|null`, 32 Bytes - the receipts root of the most recent *valid* block in the branch defined by payload and its ancestors
```

Comment:
@karalabe could you please share more info about why `latestValidHash` can not work in the case of stateless execution?

Reply:
There's no local chain. Stateless execution is stateless. Here's the input, here's the output. Forget we ever talked.

Reply:
It's more akin to a library verifying an execution than a full node being on the other side.